sdk/servicebus/service-bus/karma.conf.js

```diff
@@ -69,7 +69,17 @@ module.exports = function(config) {
       "AAD_TENANT_ID",
       "RESOURCE_GROUP",
       "AZURE_SUBSCRIPTION_ID",
-      "CLEAN_NAMESPACE"
+      "CLEAN_NAMESPACE",
+      "MANAGEMENT_QUEUE_1_BROWSER",
+      "MANAGEMENT_TOPIC_1_BROWSER",
+      "MANAGEMENT_SUBSCRIPTION_1_BROWSER",
+      "MANAGEMENT_RULE_1_BROWSER",
+      "MANAGEMENT_TOPIC_2_BROWSER",
+      "MANAGEMENT_SUBSCRIPTION_2_BROWSER",
+      "MANAGEMENT_TOPIC_3_BROWSER",
+      "MANAGEMENT_SUBSCRIPTION_3_BROWSER",
+      "MANAGEMENT_NEW_ENTITY_1_BROWSER",
+      "MANAGEMENT_NEW_ENTITY_2_BROWSER"
```

Comment:
Do we expect these resources to exist before the tests are run?
Can we not use the management apis themselves in the beforeEach() section to create the required resources? If yes, then the name of the entities can remain between the beforeEach hook and the test itself, and we can skip expecting these env vars to exists

Reply:
Reason these need to be added to env vars is because we need to generate the counterparts with -browser suffix

Reply:
The counterparts with the -browser suffix can still be generated if the values of these env vars are stored as constants in the test file right?

The non management api tests needed the env variables because these entities were expected to be exist before the test run starts. But the management api tests in atomManagement.spec.ts file are written in such away that the beforeEach() hook creates the required entities. So having the entity names as constants should suffice.

Reply:
The management API tests too require the resources to be existing before the test runs. 
The non-management API tests too use the beforeEach hook.

We currently have a central/single way of creating the test resources for Node vs browser. Moving these into individual test file would be another way of achieving the same.
Seems unnecessary duplication and maybe harder for anyone coming in to work with test infrastructure to implement things if need be.

Is there any concern / problem with current state as such? These entities play same role as "PARTITIONED_QUEUE", "UNPARTITIONED_TOPIC" etc that we have. And in future would be necessary too to achieve improved isolation of test runs for devs.

Reply:
Can you share more about why we need to have resources created for the management API tests? Is this to test the  skip/top methods?

Reply:
Yes and for get and update APIs as well

Reply:
Requiring all of these upfront means a lot more work needs to be done in our setup. 

As we discussed, can we potentially update the way we write the tests to not assume a specific number of entities are present? I would rather explore that approach first before requiring 9 more things be present and in a specific state prior to the test running.

Reply:
For get() on entities for instance, testing that would require an entity to exist. 

And one of the goals we had was for ensuring developers can run the tests in isolation where possible. This is to help with development time coding efficiency etc. as was discussed earlier with @chradek @ramya-rao-a 

Is there a specific suggestion/proposal on how do we want to write the get() tests differently now?

Reply:
Also, could you clarify on the work required for setup?
I think we might be misunderstanding that Key Vault entries need to be configured for these. However, this isn't currently required of the entity names in these and all tests in the SDK.

Reply:
I am not sure I understand this. If this list is not the list of resources required to exist before the test runs, then what are they for?

Reply:
Yes, it is the list of test resources that need to exist before the test runs.
They are also reused for some testcases and so may get modified, recreated during course of the entire management API test runs.

Reply:
As described in #6716, we want to strive towards a place where we do not expect any entities to exist before starting a test run.

This PR adds a new list of entities that we expect to exist before a test run is started.

I would strongly urge to follow up on #6716 first. The refactoring done there should make it easier to enable browser tests.

Reply:
With current changes, all of the entities created in tests for the management API feature are deleted after the test runs complete.

#6716 seems to be applicable for Service Bus AMQP related tests as well. It doesn't seem like they need not be related and browser test runs can be enabled. Is there anything specific missing that needs to block the PR as such?

Reply:
> #6716 seems to be applicable for Service Bus AMQP related tests only

#6716 applies to all tests, amqp and http, node and browser.
The suggestion to tackle that first was to avoid adding a whole new set entities that we expect to exist.

But as discussed offline, we can focus on getting this PR merged first and then focus on #6716